src/node/chainstate.h

```diff
@@ -45,10 +45,10 @@ enum class ChainstateLoadingError {
  *        differentiable by the specific enumerator.
  *
  *        Note that a return value of SHUTDOWN_PROBED means ONLY that "during
- *        this sequence, when we explicitly checked ShutdownRequested() at
+ *        this sequence, when we explicitly checked shutdown_requested() at
  *        arbitrary points, one of those calls returned true". Therefore, a
  *        return value other than SHUTDOWN_PROBED does not guarantee that
- *        ShutdownRequested() hasn't been called indirectly.
+ *        shutdown_requested() hasn't been called indirectly.
```

Comment:
I think this comment is "wrong". `shutdown_requested` is a local symbol to the function, so there is no other place calling it.

Maybe just rephrase as "shutdown hasn't been requested indirectly"